src/abstractdataframe/selection.jl

```diff
@@ -67,6 +67,10 @@ const TRANSFORMATION_COMMON_RULES =
     `cols => function`. In particular the following expression `function => target_cols`
     is not a valid transformation specification.
 
+    Note! If `cols` or `target_cols` is one of `All`, `Cols`, `Between`, or
+    `Not` and is used in a broadcasting context are properly expanded to
```

Comment:
There's a syntax problem here. Just copy the text from above?

Reply:
I have forgotten to sync the docstrings. Fixed.